### PR TITLE
fix: solving pt-br translation issues

### PR DESCRIPTION
**Pull request by hugovms**
| Q             | A
| ------------- | ---
| Branch?       | 5.x for features / 4.4 or 5.1 for bug fixes 
| Bug fix?      | yes/no
| New feature?  | yes/no 
| Deprecations? | yes/no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by Nyholm** [comment]
Could I ask @adielcristo and @cenoura to review this?

File: src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf [hunk 1]
```diff
@@ -176,27 +176,27 @@
             </trans-unit>
             <trans-unit id="47">
                 <source>This value should be the user's current password.</source>
-                <target>Este valor deveria de ser a password atual do utilizador.</target>
+                <target>Este valor deveria de ser a senha atual do usuário.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf:179]:
```suggestion
                <target>Este valor deveria ser a senha atual do usuário.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf [hunk 2]
```diff
@@ -176,27 +176,27 @@
...
             </trans-unit>
             <trans-unit id="48">
                 <source>This value should have exactly {{ limit }} character.|This value should have exactly {{ limit }} characters.</source>
-                <target>Este valor tem de ter exatamente {{ limit }} carateres.</target>
+                <target>Este valor deve possuir exatamente {{ limit }} carateres.</target>
             </trans-unit>
             <trans-unit id="49">
                 <source>The file was only partially uploaded.</source>
-                <target>Só foi enviado parte do ficheiro.</target>
+                <target>Só foi enviado uma parte do arquivo.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf:187]:
```suggestion
                <target>Só foi enviada uma parte do arquivo.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf [hunk 3]
```diff
@@ -176,27 +176,27 @@
...
             </trans-unit>
             <trans-unit id="50">
                 <source>No file was uploaded.</source>
-                <target>Nenhum ficheiro foi enviado.</target>
+                <target>Nenhum arquivo foi enviado.</target>
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Não existe nenhum directório temporária configurado no ficheiro php.ini.</target>
+                <target>Não existe nenhuma pasta temporária configurada no arquivo do php.ini.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf:195]:
```suggestion
                <target>Não existe nenhuma pasta temporária configurada no arquivo php.ini.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf [hunk 4]
```diff
@@ -292,15 +292,15 @@
             </trans-unit>
             <trans-unit id="76">
                 <source>The image is landscape oriented ({{ width }}x{{ height }}px). Landscape oriented images are not allowed.</source>
-                <target>A imagem está orientada à paisagem ({{ width }}x{{ height }}px). Imagens orientadas à paisagem não são permitidas.</target>
+                <target>A imagem está em orientação de paisagem com ({{ width }}x{{ height }}px). Imagens orientadas em paisagem não são permitidas.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf:295]:
```suggestion
                <target>A imagem está em orientação de paisagem com {{ width }}x{{ height }}px. Imagens orientadas em paisagem não são permitidas.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf [hunk 5]
```diff
@@ -292,15 +292,15 @@
...
             </trans-unit>
             <trans-unit id="77">
                 <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
-                <target>A imagem está orientada ao retrato ({{ width }}x{{ height }}px). Imagens orientadas ao retrato não são permitidas.</target>
+                <target>A imagem está em orientação de retrato com ({{ width }}x{{ height }}px). Imagens orientadas em retrato não são permitidas.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf:299]:
```suggestion
                <target>A imagem está em orientação de retrato com {{ width }}x{{ height }}px. Imagens orientadas em retrato não são permitidas.</target>
```
With "com" without (), or with () without "com".

File: src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf (hunk 3 above)
Comment (cenoura) [reply src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf:195]: I'd suggest "Não existe uma pasta temporária configurada no arquivo php.ini." so it doesn't repeat the negation

**Review by cenoura — CHANGES_REQUESTED** [review]

**Comment by fabpot** [comment]
@hugovms Can you take comments into accounts? Thank you.

**Comment by fabpot** [comment]
Anyone willing to finish this PR?

**Comment by nicolas-grekas** [comment]
Thank you @hugovms, this is now part of #39197.

**Pull request merged**